modules/framework/release.yaml

```diff
@@ -269,6 +269,15 @@ api:
     isolated:     true
     codecoverage: true
 
+  - artifact: dev.galasa.framework.api.streams
+    version: 0.40.0
```

Comment:
This will need to be bumped up to 0.41.0 - try merging the latest from main and then rebuilding